src/main/kotlin/no/nav/helse/fritakagp/Env.kt

```diff
@@ -55,6 +57,24 @@ sealed class Env private constructor(
     val tokenExchangeEndpoint = "auth.token_exchange_endpoint".prop()
     val tokenIntrospectionEndpoint = "auth.token_introspection_endpoint".prop()
 
+    val idportenConfig =
+        TokenSupportConfig(
+            IssuerConfig(
+                name = "idporten-issuer",
+                discoveryUrl = "idporten_config.discoveryurl".prop(),
+                acceptedAudience = "idporten_config.accepted_audience".prop().let(::listOf)
+            )
+        )
+
+    val tokenxConfig =
+        TokenSupportConfig(
+            IssuerConfig(
+                name = "tokenx-issuer",
```

Comment:
Variabel for issuer-navnene? Kan da brukes i `FritakModule.kt`.

Reply:
det skal vi fikse 👍